Libraries/Text/Text/RCTTextView.m

```diff
@@ -533,8 +564,6 @@ - (void)copy:(id)sender
   UIPasteboard *pasteboard = [UIPasteboard generalPasteboard];
   pasteboard.items = @[item];
 #elif TARGET_OS_OSX
-  [_textView copy:sender];
-
```

Comment:
We don't need to forward the copy event to NSTextView since it will be the first responder when clicked. The copy event will be handled by the NSTextView.


https://user-images.githubusercontent.com/96719/202544061-22faa4c5-9fc8-44e9-be6e-904609e41a08.mp4